html/beps/bep_encrypted_data.rst

```diff
@@ -73,16 +73,16 @@ Metadata format
   The protocol version used to encrypt the torrent, currently *1*. New versions may be introduced by updates to this BEP if cryptographic weaknesses necessitate incompatible changes. Implementations should check if they support the version indicated in the metadata file and otherwise inform the user that they can download the data but not decrypt it.
 
 ``shadow``
-  bencoded-then-encrypted dictionary of key-value pairs that shadow entries in the info dictionary.
-  If it is absent only the payload is encrypted and all info dictionary entries are non-shadowed.
-  Implementations should only shadow a whitelist of keys which they have a shadowing strategy and ignore other keys.
-  Shadowable keys suggested by this BEP: ``length, files, name, comment``.
+  bencoded-then-encrypted dictionary whose key-value pairs shadow entries in the info dictionary.
+  If it is absent only the payload is encrypted and no info dictionary entries are shadowed.
+  Implementations should only shadow a whitelist of keys for which they have a shadowing strategy and ignore other keys.
+  Shadowable keys suggested by this BEP: ``length``, ``files``, ``name``, ``comment``.
 
 ``mac``
   message authentication code covering the info dictionary
 
 ``name``
-  the name field is a mandatory part of [BEP 3]_, therefore a placeholder MUST be provided if a shadow name is used. An implementation may either generate a random string consisting of filesystem-friendly characters or allow the user to choose a public name that reveals less information than the shadow name.
+  the name field is a mandatory part of [BEP 3]_. If a shadow name is used then a placeholder MUST be provided. An implementation may either generate a random string consisting of filesystem-friendly characters or allow the user to choose a public name that reveals less information than the shadow name.
```

Comment:
I see, this is what you meant. Let's take an example. uTorrent put some audio and video metadata in torrents. I believe these fields would be quite revealing and should be encrypted. what is the purpose of providing placeholders for them? and what are the expectations on those placeholders, does the entire bencoded sub-tree need to be there? (presumably not, because dictionary keys could contain revealing information too). I think it might make sense to motivate this requirement and describe it in some more details (or drop it)